code/game/objects/items/weapons/implants/implantfluff.dm

```diff
@@ -0,0 +1,114 @@
+
+/**
+ * # Implant Fluff Datum
+ *
+ * All implants have used to have a get_data() call which used to return html, this datum serves to better method to
+ * store this data and allow it to actually be accessed and used seperately from browserUI. Implant fluff is just lore
+ * about the implant that is accessed through the implantpad, you must attach one of these datums to the implant_data
+ * var on an implant for it to show it up.
+ */
+/datum/implant_fluff
+	var/name = "Unknown Implant"
+	var/life = "Not Specified"
+	var/notes = "No Notes"
+	var/function = "An electronic insert that supplements biological functions"
+
+/datum/implant_fluff/abductor
+	name = "GC$X9%D#G"
+	life = "Unknown"
+	notes = "Appears to contain highly advanced bluespace technology"
```

Comment:
I think it's slightly better to have these fluff datums right next to corresponding implant, rather than in this one file. But it's whatever